src/main/java/uk/gov/hmcts/ccd/v2/external/controller/CaseController.java

```diff
@@ -437,4 +448,89 @@ public CaseController(
         final CaseDetails caseDetails = createCaseOperation.createCaseDetails(caseTypeId, content, ignoreWarning);
         return status(HttpStatus.CREATED).body(new CaseResource(caseDetails, content, ignoreWarning));
     }
+
+    @GetMapping(
+        path = "getLinkedCases/{caseReference}",
+        produces = {
+            V2.MediaType.CASE
+        }
+    )
+    @ApiOperation(
+        value = "Retrieve Linked Cases"
+    )
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = "Success",
+            response = CaseResource.class),
+        @ApiResponse(
+            code = 400,
+            message = V2.Error.CASE_ID_INVALID),
```

Comment:
As there are multiple 400 reasons, replace with : 
```java
        @ApiResponse(
            code = 400,
            message = "One or more of the following reasons:"
                + "\n1) " + V2.Error.CASE_ID_INVALID
                + "\n2) " + V2.Error.PARAM_NOT_NUM,
```

Reply:
Fixed in [54f1d4f](https://github.com/hmcts/ccd-data-store-api/pull/1920/commits/54f1d4fcf3faac512d1a6278a71e82b99881cce4)